### PR TITLE
circleci: add failing circle config

### PR DESCRIPTION
**Pull request by tmc**

### CODE REVIEW

**Comment by tmc** [comment]
test comment

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/go
+    steps:
```
Comment (tmc) [inline .circleci/config.yml:6]: this is a test code comment

**Comment by tmc** [comment]
@%v, it looks like there was a test failure, can you please investigate?

**Comment by tmc** [comment]
@%v, it looks like there was a test failure, can you please investigate?

**Comment by tmc** [comment]
@github.User{Login:"tmc", ID:3977, AvatarURL:"https://avatars1.githubusercontent.com/u/3977?v=4", HTMLURL:"https://github.com/tmc", GravatarID:"", Type:"User", SiteAdmin:false, URL:"https://api.github.com/users/tmc", EventsURL:"https://api.github.com/users/tmc/events{/privacy}", FollowingURL:"https://api.github.com/users/tmc/following{/other_user}", FollowersURL:"https://api.github.com/users/tmc/followers", GistsURL:"https://api.github.com/users/tmc/gists{/gist_id}", OrganizationsURL:"https://api.github.com/users/tmc/orgs", ReceivedEventsURL:"https://api.github.com/users/tmc/received_events", ReposURL:"https://api.github.com/users/tmc/repos", StarredURL:"https://api.github.com/users/tmc/starred{/owner}{/repo}", SubscriptionsURL:"https://api.github.com/users/tmc/subscriptions"}, it looks like there was a test failure, can you please investigate?

**Comment by tmc** [comment]
@tmc, it looks like there was a test failure, can you please investigate?

**Comment by tmc** [comment]
@tmc, it looks like there was a test failure, can you please investigate?

**Comment by tmc** [comment]
@tmc, it looks like there was a test failure, can you please investigate?

**Comment by tmc** [comment]
@tmc, it looks like there was a test failure, can you please investigate?

**Comment by redshiftzero** [comment]
hehe ^^^^!@!!! @tmc

**Review by redshiftzero — CHANGES_REQUESTED** [review]
Major problems here. This is a test repo for one and this is a test review to check the bot Does a Good Thing

**Pull request closed without merging**